query-engine/connectors/sql-query-connector/src/database/sqlite.rs

```diff
@@ -68,6 +70,12 @@ impl FromSource for Sqlite {
         builder.test_on_check_out(true);
 
         let pool = builder.build();
+        pool.check_out().await.map_err(|err| {
+            ConnectorError::from_kind(ErrorKind::InvalidDatabaseUrl {
+                details: err.to_string(),
+                url: database_str.to_string(),
+            })
```

Comment:
I assume this is for sqlite — could another error happen at this point? It would be confusing to return an invalid database error for something else

Reply:
Ah, yeah I forgot this map, I'm removing it and SQLite users can lick their wounds.